### PR TITLE
Issue/5438 add prettier 3.0.x

### PR DESCRIPTION
**Pull request by gcusnieux**
fix gravitee-io/issues#5438

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -185,8 +186,11 @@
...
     "buildNoReg": "gulp buildNoReg",
-    "lint": "gulp lint",
-    "lint:fix": "gulp lint:fix",
-    "compile": "gulp compile"
+    "lint": "gulp lint && npm run prettier",
+    "lint:fix": "gulp lint:fix && npm run prettier:fix",
+    "compile": "gulp compile",
+    "prettier": "prettier --check \"**/*.{ts,html,scss,json}\"",
+    "prettier:list": "prettier --list-different \"**/*.{ts,html,scss,json}\"",
+    "prettier:fix": "prettier --write \"**/*.{ts,html,scss,json}\""
```
Comment (phiz71) [inline package.json:194]: I think we also have `css` and `js` files
```suggestion
    "prettier": "prettier --check \"**/*.{js,ts,html,css,scss,json}\"",
    "prettier:list": "prettier --list-different \"**/*.{js,ts,html,css,scss,json}\"",
    "prettier:fix": "prettier --write \"**/*.{ts,html,cs,scss,json}\""
```

File: tslint.json [hunk 2]
```diff
@@ -43,36 +27,15 @@
...
-      "check-catch",
-      "check-else",
-      "check-whitespace"
-    ],
-    "quotemark": [true, "single"],
+    "one-line": [true, "check-open-brace", "check-catch", "check-else", "check-whitespace"],
+    "quotemark": [true, "single", "avoid-escape", "avoid-template"],
     "radix": true,
-    "semicolon": true,
+    "semicolon": false,
```
Comment (phiz71) [inline tslint.json:33]: why ?

Comment (gcusnieux) [reply tslint.json:33]: It's managed by prettier now

File: src/management/api/portal/plans/plan/plan-wizard-policies.html [hunk 3]
```diff
@@ -80,27 +100,27 @@
...
+          <md-button class="md-raised md-warn" ui-sref="management.apis.detail.portal.plans.list">CANCEL</md-button>
         </div>
         <div flex layout="row" layout-align="end top">
-          <md-button
-            ng-disabled="$ctrl.policyConfigurationForm.$invalid"
-            ng-click="$ctrl.gotoNextStep()"
-            class="md-primary md-raised">SAVE</md-button>
+          <md-button ng-disabled="$ctrl.policyConfigurationForm.$invalid" ng-click="$ctrl.gotoNextStep()" class="md-primary md-raised"
+            >SAVE</md-button
+          >
```
Comment (phiz71) [inline src/management/api/portal/plans/plan/plan-wizard-policies.html:123]: It seems that the formatter has a strange behavior here.

File: src/management/api/proxy/backend/healthcheck/healthcheck-configure.html [hunk 4]
```diff
@@ -18,41 +18,48 @@
...
+              >
                 Inherit configuration
-                <span class="ipsum" ng-if="healthCheckCtrl.rootHealthcheckEnabled">Inherit health-check configuration from global health-check configuration.</span>
-                <span class="ipsum" ng-if="!healthCheckCtrl.rootHealthcheckEnabled">You must enable a global health-check to be able to inherit from it.</span>
+                <span class="ipsum" ng-if="healthCheckCtrl.rootHealthcheckEnabled"
+                  >Inherit health-check configuration from global health-check configuration.</span
+                >
+                <span class="ipsum" ng-if="!healthCheckCtrl.rootHealthcheckEnabled"
+                  >You must enable a global health-check to be able to inherit from it.</span
+                >
```
Comment (phiz71) [inline src/management/api/proxy/backend/healthcheck/healthcheck-configure.html:62]: once again, strange behavior regarding the `>` character

**Pull request merged**